easybuild/easyconfigs/i/ImageMagick/ImageMagick-6.9.3-2-goolf-1.4.10.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'ImageMagick'
+version = '6.9.3-2'
```

Comment:
I found this, but not all versions are there: http://www.imagemagick.org/download/releases/